components/grid/editing/overview.md

```diff
@@ -44,6 +44,8 @@ You can initiate editing or inserting of an item from anywhere on the page (butt
 
 ## Example
 
+>tip You can see the CRUD events in action in our live demos for [Inline](https://demos.telerik.com/blazor-ui/grid/editing-inline), [Popup](https://demos.telerik.com/blazor-ui/grid/editing-popup) and [InCell](https://demos.telerik.com/blazor-ui/grid/editing-incell) editing. You can also use the [Telerik Wizard]({%slug getting-started-vs-integration-new-project%}) project templates to easily create an application that showcases the Telerik Grid with CRUD events implemented.
```

Comment:
Does moving this after the paragraph look better (I think it would)?

Reply:
Indeed, moved below the paragraph.